README.md

```diff
@@ -1 +1,26 @@
-# halcyonsim
\ No newline at end of file
+# HalcyonSim
+
+Simulates a virtual world environment. With sun, moon, clouds, land, buildings *(objects with seamless textures to give the desired look)*, avatars.. and interactive objects that are scripted.
+
+HalcyonGrid or HalcyonSim was a fork from the <a href="http://opensimulator.org" target="_blank" rel="nofollow" >OpenSimulator</a> Project in 2010, 
+
+## Virtual World where people may move about in a full 3D world.
+
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D Graphics Application, commonly known as a "Viewer," which is used to access these virtual worlds. (not a VR headset) such as the most common one, <a href="https://firestormviewer.org" target="_blank" rel="follow">FirestormOS</a>.  *(the "OS" standing for OpenSim, and not Operating System)* also it is similar to <a href="https://SecondLife.com" target="_blank" rel="nofollow" >Second Life®</a>.  It allows users "talk" to other people in-world through nearby chat, IM's, group chats and group IM's and sometimes in voice, depending on how the grid is setup and the permissions set on individual regions. Although that's as much like a game as it gets. There is also the ability to build objects in-world known as primatives or prims in various shapes and sizes to make different types of 3D objects. Additionally other 3D items known as "Mesh" (DEA files) can be imported which have been created/edited with a 3D object editor/creator such as the free program <a href="https://store.steampowered.com/app/365670/Blender/" target="_blank" rel="follow" >"Blender"</a>.
```

Comment:
Double spaces should be single spaces.
"Graphics Application" is not a title and thus should not be capitalized.
Commas go outside of quotes, which are also unneeded in this case.
Several run-on sentences here.

Reply:
The quotes I mentioned are those that are being used like the commonly misused air quotes in speech. Some examples:
- `commonly known as a "Viewer,"` could simply be `commonly known as a viewer,` without any semantic loss.
- `It allows users "talk" to` likewise could be `It allows users to talk to` - with the additional detail of fixing the grammar with the additional `to`. If you are concerned that the word "talk" contains too much semantic meaning of verbal communication, then simply choose another word that doesn't convey that meaning. For me the word is fine, it just doesn't need the quotes.

I hope that from those few examples you can extrapolate the kind of change requested to the rest of the prose.

Additionally I see `IM's` - that's a possessive form being used as a plural.